validation/src/main/scala/hmda/validation/rules/lar/validity/V230.scala

```diff
@@ -0,0 +1,16 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ LoanApplicationRegister }
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+
+object V230 extends EditCheck[LoanApplicationRegister] {
+
+  val occupancyStatusTypes = List(1, 2, 3)
```

Comment:
Small comment: I don't think "Types" is an accurate way to name this list. Maybe `occupancyStatuses` or `occupancyValues` instead? Or you could use the list directly inline on line 12 so you don't have to name it.